views/species.erb

```diff
@@ -1,14 +1,153 @@
 <!DOCTYPE html>
 <html>
-    <head>
-      <link rel="stylesheet" type="text/css" href="/stylesheets/sass/style.css" />
-      <title>Species View</title>
-    </head>
-    <body>
-      <div class='test'>
-        <% @rows.each do |row| %>
-          <%= row %>
-        <% end %>
+  <head>
```

Comment:
Can we try to use partials please?
I feel this file is too long and difficult to read.
By using partials we could enclose relevant sections into a separate file for better readability; for example, the `head` tag could be in it's own file, as well as the main gray box div and the lists as well.

The lists partials could potentially be reused as well, as only the content should change

Reply:
Ah thanks, should I try something like this? http://recipes.sinatrarb.com/p/helpers/partials?

Reply:
Uhm I found an easier way..
```
<%= erb :mypartial %>
```
Sinatra is cool!